src/components/transactions/ExecuteTxButton/index.tsx

```diff
@@ -31,19 +33,41 @@ const ExecuteTxButton = ({
     setOpen(true)
   }
 
+  const onMouseOver = () => {
+    setWillExecute(txSummary.id)
+  }
+
+  const onMouseLeave = () => {
+    setWillExecute(undefined)
+  }
+
   return (
     <>
       <Track {...TX_LIST_EVENTS.EXECUTE}>
         {compact ? (
           <Tooltip title="Execute" arrow placement="top">
             <span>
-              <IconButton onClick={onClick} color="primary" disabled={isDisabled} size="small">
+              <IconButton
+                onClick={onClick}
+                onMouseOver={onMouseOver}
```

Comment:
`onMouseOver` fires several times when hovering. You probably want `onMouseEnter`.